app/components/preprint-form-authors.js

```diff
@@ -10,26 +10,158 @@ export default CpPanelBodyComponent.extend(PreprintFormFieldMixin, {
+                this.set('addState', 'searchView');
+            }
+        },
+        // Removes contributor then redraws contributor list view - removal of contributor may change
+        // which additional update/remove requests are permitted.
         removeContributor(contrib) {
             this.sendAction('removeContributor', contrib);
+            this.toggleProperty('removalToggle');
+            this.toggleProperty('permissionToggle');
+            this.toggleProperty('bibliographicToggle');
+            this.removedSelfAsAdmin(contrib, contrib.get('permission'));
+            this.get('contributors').removeObject(contrib);
         },
-        permissionChange(contributor, permission) {
+        // Updates contributor then redraws contributor list view - updating contributor
+        // permissions may change which additional update/remove requests are permitted.
+        updatePermissions(contributor, permission) {
             this.set(`permissionChanges.${contributor.id}`, permission.toLowerCase());
+            this.sendAction(
+                'editContributors',
+                this.get('contributors'),
+                this.get('permissionChanges'),
+                {}
+            );
+            this.set('permissionChanges', {});
+            this.toggleProperty('permissionToggle');
+            this.toggleProperty('removalToggle');
+            this.removedSelfAsAdmin(contributor, permission);
+
         },
-        bibliographicChange(contributor, isBibliographic) {
+        // Updates contributor then redraws contributor list view - updating contributor
+        // bibliographic info may change which additional update/remove requests are permitted.
+        updateBibliographic(contributor, isBibliographic) {
             this.set(`bibliographicChanges.${contributor.id}`, isBibliographic);
-        },
-        updateContributors() {
             this.sendAction(
                 'editContributors',
                 this.get('contributors'),
-                this.get('permissionChanges'),
+                {},
                 this.get('bibliographicChanges')
             );
+            this.set('bibliographicChanges', {});
+            this.toggleProperty('bibliographicToggle');
+            this.toggleProperty('removalToggle');
+        },
+        // There are 3 view states on left side of Authors panel.  This switches to add unregistered contrib view.
+        unregisteredView() {
+            this.set('addState', 'unregisteredView');
+        },
+        // There are 3 view states on left side of Authors panel.  This switches to searching contributor results view.
+        searchView() {
+            this.set('addState', 'searchView');
+            this.set('fullName', '');
+            this.set('email', '');
+        },
+        emptyView() {
+            this.set('addState', 'emptyView');
+            this.set('searchQuery', '');
+            Ember.$('.searchQuery')[0].value = '';
```

Comment:
I just wasn't sure how to access the value of the search box here.  When I use an input helper I can get the value of it easily but if I don't use a helper I can't figure out how to access it without jquery.